<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZXBWindow</class>
 <widget class="QMainWindow" name="ZXBWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1032</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TDCS/CTC3.0通用车站操作终端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:vertical {
 	width: 16px;
	background-color: rgb(222, 226, 229);
}

QScrollBar::handle:vertical {
	background-color: rgba(160, 160, 160, 240);
	min-height: 24px;
	border-radius:3px; 
	margin: 10px 5px 10px 5px;
}

QScrollBar::handle:vertical:hover,
QScrollBar::handle:vertical:pressed {
	background-color: rgba(160, 160, 160, 240);
	min-height: 24px;
 	border-radius: 3px;
	margin: 10px 5px 10px 5px;
}

QScrollBar::sub-page:vertical {
      background-color: rgb(222, 226, 229);
}

QScrollBar::add-page:vertical {
      background-color: rgb(222, 226, 229);
}

QScrollBar::sub-line:vertical {
      border: none;
}

QScrollBar::add-line:vertical {
      border: none;
}


QScrollBar:horizontal {
 	height: 16px;
	background-color: rgb(222, 226, 229);
}

QScrollBar::handle:horizontal {
	background-color: rgba(160, 160, 160, 240);
	border-radius:3px; 
	margin: 5px 10px 5px 10px;
}

QScrollBar::handle:horizontal:hover,
QScrollBar::handle:horizontal:pressed {
	background-color: rgba(160, 160, 160, 240);
 	border-radius: 3px;
	margin: 5px 10px 5px 10px;
}

QScrollBar::sub-page:horizontal {
	background-color: rgb(222, 226, 229);
}

QScrollBar::add-page:horizontal {
	background-color: rgb(222, 226, 229);
}

QScrollBar::sub-line:horizontal  {
      border: none;
}

QScrollBar::add-line:horizontal  {
      border: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 226, 229);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/res/T3.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TDCS/CTC3.0通用车站操作终端</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(240, 240, 240);
	border:none;
	image: url(:/res/Recovery1.png);
}
QPushButton:hover{
	image: url(:/res/Recovery2.png);
}
QPushButton:pressed{
	image: url(:/res/Recovery3.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(240, 240, 240);
	border:none;
	image: url(:/res/MaxWnd1.png);
}
QPushButton:hover{
	image: url(:/res/MaxWnd2.png);
}
QPushButton:pressed{
	image: url(:/res/MaxWnd3.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(240, 240, 240);
	border:none;
	image: url(:/res/CloseWnd1.png);
}
QPushButton:hover{
	background-color: rgb(200, 40, 30);
	image: url(:/res/CloseWnd2.png);
}
QPushButton:pressed{
	background-color: rgb(180, 40, 35);
	image: url(:/res/CloseWnd2.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tittleLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(0, 85, 255);
border-bottom:2px solid rgb(160, 160, 160);</string>
      </property>
      <property name="text">
       <string>进路控制台</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="ZXBTrackWidget" name="widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1918</width>
         <height>724</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ZXBRouteWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>240</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ZXBShuntWidget" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>240</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ZXBFLWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>240</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZXBRouteWidget</class>
   <extends>QWidget</extends>
   <header location="global">ZXBWindow/zxbroutewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZXBFLWidget</class>
   <extends>QWidget</extends>
   <header>ZXBWindow/zxbflwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZXBTrackWidget</class>
   <extends>QWidget</extends>
   <header>ZXBWindow/zxbtrackwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZXBShuntWidget</class>
   <extends>QWidget</extends>
   <header location="global">ZXBWindow/zxbshuntwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
